src/components/AreaView/AreaView.jsx

```diff
@@ -1,15 +1,13 @@
 import { React, useState, useEffect } from 'react';
 import { Link, Navigate, useParams } from 'react-router-dom';
-import { Button, Card } from 'react-bootstrap';
+import { Button, Card, DropdownButton, Dropdown } from 'react-bootstrap';
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Button' is defined but never used.